website/content/en/docs/crds-scope.md

```diff
@@ -0,0 +1,80 @@
+---
+title: CRD scope with Operator SDK
+linkTitle: CRD Scope
+weight: 60
+---
+
+## Overview
+
+The CustomResourceDefinition (CRD) scope can also be changed for cluster-scoped operators so that there is only a single instance (for a given name) of the CRD to manage across the cluster.
```

Comment:
Can you break up lines in this doc, ex:
```suggestion
The CustomResourceDefinition (CRD) scope can also be changed for cluster-scoped operators
so that there is only a single instance (for a given name) of the CRD to manage across the cluster.
```

Reply:
This content is not part of the scope. It was just moved. 
But OK.